thoth/storages/graph/dgraph.py

```diff
@@ -2160,18 +2205,18 @@ def sync_adviser_result(self, document: dict) -> None:
                         )
                     overall_score = entry["overall_score"]
 
-            if result[1] and result[1].get("requirements_locked"):
-                advised_software_stack = self.create_advised_software_stack_pipfile(
-                    adviser_document_id,
-                    (result[1] or {}).get("requirements_locked") or [],
-                    advised_stack_index=idx,
-                    performance_score=performance_score,
-                    overall_score=overall_score,
-                    runtime_environment=runtime_environment,
-                )
-                Advised.from_properties(source=adviser_run, target=advised_software_stack).get_or_create(
-                    self.client
-                )
+                if result[1] and result[1].get("requirements_locked"):
+                    advised_software_stack = self.create_advised_software_stack_pipfile(
+                        adviser_document_id,
+                        (result[1] or {}).get("requirements_locked") or [],
+                        advised_stack_index=idx,
+                        performance_score=performance_score,
+                        overall_score=overall_score,
+                        run_software_environment=run_software_environment,
+                    )
+                    Advised.from_properties(source=adviser_run, target=advised_software_stack).get_or_create(
+                        self.client
+                    )
```

Comment:
This looks like wrong rebase of https://github.com/thoth-station/storages/commit/af670c90fc009846f17d39a51137edb7aa469880?

Reply:
Correct, solved. Thanks!!